### PR TITLE
Migrate landing to typescript

### PR DESCRIPTION
**Pull request by paradus-hex**

### CODE REVIEW

File: src/components/AboutUs.tsx [hunk 1]
```diff
@@ -1,20 +1,19 @@
...
             </h3>
-            <p class="m-4  text-2xl">
+            <p className="m-4  text-2xl">
               We are four friends, all of us completing our undergraduate,
               studying different majors but united by a common frustration with
               the current education system. It felt like the tools that we
               needed to truly understand and retain our materials were totally
               absent, even after scouring through the whole internet. And we
-              weren't alone. Our classmates felt the same way.
+              weren't alone. Our classNamemates felt the same way.
```
Comment (imoxto) [inline src/components/AboutUs.tsx:16]: rename

File: src/components/Features.tsx [hunk 2]
```diff
@@ -1,25 +1,22 @@
 import { featureSection } from '../constants/constants.js';
```
Comment (arionrefat) [inline src/components/Features.tsx:1]: It's not a js file anymore

Comment (arionrefat) [inline src/components/Features.tsx:1]: Update the `constants.js`

Comment (arionrefat) [inline src/components/Features.tsx:1]: change the `constants.js`

**Review by arionrefat — CHANGES_REQUESTED** [review]

**Pull request merged**